browser-sign-in/OktaBrowserSignIn/WelcomeViewController.swift

```diff
@@ -35,13 +38,51 @@ final class WelcomeViewController: UIViewController {
         }
     }
     
-    func navigateToDetailsPage() {
-        guard Credential.default != nil else { return }
-        performSegue(withIdentifier: "show-details", sender: self)
+    func SignInWithBiometrics() {
```

Comment:
```suggestion
    func signInWithBiometrics() {
```

Functions should not start with an upper-case letter.